### PR TITLE
Alex krasn/receipt demo featture

### PR DESCRIPTION
**Pull request by alex-krasn**
first commit merging with changes on master and integrating them

### CODE REVIEW

File: src/react/components/pages/predict/predictPage.tsx [hunk 1]
```diff
@@ -339,10 +430,48 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                     when={this.state.isPredicting}
                     message={"A prediction operation is currently in progress, are you sure you want to leave?"}
                 />
-            </div>
+            </div >
```
Comment (tianxiangs) [inline src/react/components/pages/predict/predictPage.tsx:433]: I think we dont need to have a space here

Comment (alex-krasn) [reply src/react/components/pages/predict/predictPage.tsx:433]: sure

File: src/react/components/pages/predict/predictPage.tsx [hunk 2]
```diff
@@ -199,8 +244,50 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                             <div className="alight-vertical-center mt-2">
                                 <div className="seperator"/>
                                 or
-                                <div className="seperator"/>
+                                <div className="seperator" />
```
Comment (tianxiangs) [inline src/react/components/pages/predict/predictPage.tsx:247]: also here

File: src/react/components/pages/predict/predictPage.tsx (hunk 1 above)
Comment (alex-krasn) [reply src/react/components/pages/predict/predictPage.tsx]: done

File: src/react/components/pages/predict/predictPage.tsx (hunk 2 above)
Comment (alex-krasn) [reply src/react/components/pages/predict/predictPage.tsx]: done

**Pull request merged**